src/controllers/userController.js

```diff
@@ -85,14 +76,56 @@ const login = async (req, res) => {
 
   const data = {
     user: {
+      id: foundUser.id,
       email: foundUser.email,
       token: authHelper.generateToken({ id: foundUser.id }),
-      username: foundUser.username,
       bio: foundUser.bio
     }
   };
   return successResponse(res, 200, data);
 };
 
+/**
+ * @description updates the profile of a user
+ * @param {object} request express request object
+ * @param {object} response express response object
+ * @param {object} next express next argument
+ * @returns {json} json
+ */
+const editProfile = async (request, response) => {
+  const { body, userData } = request;
+  const {
+    email, firstName, lastName, avatarUrl
+  } = body;
+  if (!email && !firstName && !lastName && !avatarUrl) {
+    return responseMessage(response, 400, { error: 'the field(s) entered are invalid' });
+  }
+  if (email) {
+    const existingUser = await User.findOne({ where: { email } });
+    if (existingUser) return responseMessage(response, 409, { error: 'email already exist' });
```

Comment:
I've seen this a couple of times, can you write your if statement with the braces?

Reply:
Noted, would add the braces